EIPS/eip-897.md

```diff
@@ -6,7 +6,7 @@ type: Standards Track
 category: ERC
 status: Draft
 created: 2018-02-21
-discussions-to: https://github.com/ethereum/EIPs/pulls/897
+discussions-to: https://github.com/ethereum/EIPs/pull/897
```

Comment:
There is a rule that this cannot point to pull requests. While 1820 is Final, it may be too late to change it there, but for this one it may be possible?

Reply:
Hm, what do you think about merging this fix anyway and posting a comment in #897 that it should be changed?